src/db.ts

```diff
@@ -28,42 +51,76 @@ interface InternalReminder {
 	ping: string | null;
 }
 
-// -- Activity Tracker Database --
-export const activityTrackerBlacklistDb = new Enmap<string[]>("trackerBlacklist");
-activityTrackerBlacklistDb.ensure("general-user", []);
-activityTrackerBlacklistDb.ensure("general-game", []);
-
-// key: "[user id]-[game]"
-export const activityTrackerLogDb = new Enmap<ActivityLogEntry[], InternalActivityLogEntry[]>({
-	name: "trackerLog",
-	serializer: (data) => data.map((e) => ({ duration: e.duration.toISO(), date: e.date.toISO() })),
-	deserializer: (data) =>
-		data.map((e) => ({
-			duration: Duration.fromISO(e.duration),
-			date: DateTime.fromISO(e.date, { setZone: true }),
-		})),
-});
+/*
+-----------------------------
+- Activity Tracker Database -
+-----------------------------
+*/
 
-export interface ActivityLogEntry {
-	duration: Duration;
-	date: DateTime;
+/** Interface for tracker logs */
+export interface TrackerLog {
+	/** When a log got logged */
+	time: ISODate;
+	/** How long a game got played */
+	playtime: number;
+	/** User id of the User who just got logged */
+	userid: userID;
+	/** Game id of the game which just got logged */
+	gameName: string;
 }
 
-interface InternalActivityLogEntry {
-	duration: ISODuration;
-	date: ISODate;
+/** Interface for stripped down tracking logs */
+export interface TrackerSublog {
+	/** User- or Game- ID */
+	id: userID;
+	/** How often a User or Game got logged */
+	logs: number;
+	/** How long a User played a certain game
+	 * (Or the other way around)
+	 */
+	playtime: number;
```

Comment:
Should be a DateTime.duration

Reply:
I wanna leave it as number in ms.
At very least I added in the doc string a note that it is in ms now

Reply:
Duration makes it easier to work with and makes things consistent across the whole codebase. No need for ugly calculations. We can just let the library handle that for us.
I can implement it myself don't worry

Reply:
I like the "raw math" and there arent that many things which are done with the playtime

I still dont see the real benefits for using such Object over just a plain old number which primarily gets used to only be converted into a timestamp